src/Sylius/Behat/Context/Api/Shop/CartContext.php

```diff
@@ -310,6 +311,48 @@ public function thisItemShouldHaveCode(array $item, string $variantCode): void
         );
     }
 
+    /**
+     * @Then /^(product "[^"]+") price should be decreased by ("[^"]+")$/
+     */
+    public function itsPriceShouldBeDecreasedBy(ProductInterface $product, int $amount): void
+    {
+        /** @var ChannelPricingInterface $pricing */
+        $pricing = $product->getVariants()->get(0)->getChannelPricingForChannel($this->sharedStorage->get('channel'));
+
+        $items = $this->responseChecker->getValue($this->cartsClient->getLastResponse(), 'items');
+
+        foreach ($items as $item) {
+            if ($item['productName'] === $product->getName()) {
+                Assert::same($pricing->getPrice() - $amount, $item['subtotal']);
+            }
+
+            return;
+        }
+
+        throw new \InvalidArgumentException('Expected product does not exist');
```

Comment:
if you create private function `isDecreasedBy(ProductInterface $product, int amount = 0): bool` you can ommit this duplicated code.